storage/InsertDestination.cpp

```diff
@@ -682,8 +685,11 @@ void PartitionAwareInsertDestination::bulkInsertTuplesWithRemappedAttributes(
     // set a bit in the appropriate TupleIdSequence.
     accessor->beginIteration();
     while (accessor->next()) {
-      TypedValue attr_val = accessor->getTypedValue(attribute_map[partition_attribute_id]);
-      partition_membership[partition_scheme_header_->getPartitionId(attr_val)]
+      PartitionSchemeHeader::PartitionValues values;
+      for (const attribute_id attr_id : partition_scheme_header_->getPartitionAttributeIds()) {
+        values.push_back(accessor->getTypedValue(attr_id));
+      }
```

Comment:
I just checked the methods you mentioned, but I don't think we could use them, as we also need to get partition id in a range partition.